lm-agent-snap/README.md

```diff
@@ -0,0 +1,76 @@
+# lm-agent-snap
+The License Managar Agent bundled into a Snap.
+
+# Installation instructions
+
+For installing from the remote store, run:
+```bash
+sudo snap install license-manager-agent
+```
+
+## Basic Usage
+
+This snap requires a few configuration values to be set before it can be used. These values are:
+- base-api-url: The URL of the Jobbergate API server where the agent will send its data. Setting/unsetting this value is more interesting when using the snap in a development environment; do not change it otherwise.
+
+- oidc-domain: The domain of the OIDC server that the agent will use for authentication. Setting/unsetting this value is more interesting when using the snap in a development environment; do not change it otherwise.
+
+- oidc-client-id: The client ID of the OIDC application that the agent will use for authentication.
+
+- oidc-client-secret: The client secret of the OIDC application that the agent will use for authentication.
+
+- task-jobs-interval-seconds: The interval in seconds at which the agent will run its internal task jobs, hence sending data to the Jobbergate API server. This is optional and defaults to 30 seconds.
+
+- task-self-update-interval-seconds: The interval in seconds at which the agent will check for updates to itself. This is optional and defaults to 30 seconds.
+
+- sbatch-path: The absolute path to the *sbatch* command on the host system. This is optional and defaults to /usr/bin/sbatch.
+
+- scontrol-path: The absolute path to the *scontrol* command on the host system. This is optional and defaults to /usr/bin/scontrol.
+
+- default-slurm-work-dir: The default working directory that the agent will use when submitting jobs to the SLURM cluster. This is optional and defaults to /tmp.
+
+- slurm-user-mapper: The user mapper that the agent will use to map the system user name to the SLURM user name. This is optional and defaults to none.
+
+- single-user-submitter: The system user name that the agent will use to submit jobs to the SLURM cluster on behalf of in case the *single-user-submitter* is not set. This is optional and defaults to *ubuntu*.
+
+- write-submission-files: A boolean value (true, false) that indicates whether the agent should write submission files to disk. This is optional and defaults to false.
```

Comment:
This needs to be re-written for the license manager config settings.

Reply:
damn, I forgot to copy from the snapcraft.yaml file

Reply:
fixed in [13c89e3](https://github.com/omnivector-solutions/license-manager/pull/389/commits/13c89e37e4fca4d4f8cd3de0090c4f227cc7a738)